zulip/zulip/cli.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+import logging
+import sys
+
+from typing import Any, Dict, List
+
+import zulip
+import click
```

Comment:
This `click` requirement is not declared in `setup.py`.

Reply:
#694